_toc.yml

```diff
@@ -0,0 +1,84 @@
+format: jb-book
+root: README
+
+parts:
+
+  - caption: Dataset Catalog
+    chapters:
+    - file: ./dataset_catalog/README
+
+
+  - caption: Environment Set-Up
+    chapters:
+    - file: ./environment_set_up/README
+
+  - caption: Dataset Preprocessing
+    chapters:
+    - file: ./dataset_preprocessing/ReChunkingData
+    - file: ./dataset_preprocessing/RechunkingData_Cloud
+    - file: ./dataset_preprocessing/archive/conus404_subset_to_zarr/readme
+    # It may be that you don't want separate sections for these sub-directories in the TOC... Delete if that's the case.
+      sections:
+      - file: ./dataset_preprocessing/archive/conus404_subset_to_zarr/daily/README   ## TODO
+      - file: ./dataset_preprocessing/archive/conus404_subset_to_zarr/hourly/README  ## TODO
+    - file: ./dataset_preprocessing/demos/README                                     ## TODO
+      # sections:
+      # - file: ./dataset_preprocessing/demos/era5-land-bitinfo
+      # - file: ./dataset_preprocessing/demos/era5-land-api_dask
+      # - file: ./dataset_preprocessing/demos/era5-land-kerchunk
+      # - file: ./dataset_preprocessing/demos/gridmet_processing_with_pynco
+    - file: ./dataset_preprocessing/tutorials/README                                 ## TODO
+      #sections:
+      #- file: ./dataset_preprocessing/tutorials/CONUS404/1_CONUS404_DRB_Data_Prep
+      #- file: ./dataset_preprocessing/tutorials/CONUS404/2_CONUS404_DRB_Analysis
+      #- file: ./dataset_preprocessing/tutorials/nwm/nwm_rechunking
+
+  - caption: Dataset Access
+    file: ./dataset_access/README                                                   ## TODO
+    chapters:
+    - file: ./dataset_access/conus404_explore
+    - file: ./dataset_access/conus404_regrid
+    - file: ./dataset_access/conus404_spatial_aggregation
+    - file: ./dataset_access/conus404_temporal_aggregation
+
+  - caption: Model Evaluation Tutorials
+    file: ./model_evaluation/tutorials/README                                 ## TODO
+    chapters:
+    - file: ./model_evaluation/tutorials/nwm_streamflow/00_Overview
+      # sections:
+      # - file: ./model_evaluation/tutorials/nwm_streamflow/01_Data_Prep
+      # - file: ./model_evaluation/tutorials/nwm_streamflow/02_Analysis_NWM
+      # - file: ./model_evaluation/tutorials/nwm_streamflow/02_Analysis_DScore
+      # - file: ./model_evaluation/tutorials/nwm_streamflow/03_Vizualization
+#   - file: ./model_evaluation/tutorials/other-tutorial-workflow/00_Overview  ## TODO
+
+
+  - caption: Docs
+    chapters:
+    - file: ./docs/ADR
+    - file: ./docs/CONTRIBUTING
+    - file: ./docs/LICENSE
```

Comment:
These need updated to `doc` - or we need to rename our folder to `docs` - I'm not sure which is more proper.